### ISSUE TITLE
Does mkdocs support Chinese search?

### ISSUE THREAD

**Issue by yuxiang660** [squidfunk/mkdocs-material#3823]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I want to suggest an idea and checked that ...

- [ ] ... to my best knowledge, my idea wouldn't break something for other users
- [ ] ... the documentation does not mention anything about my idea
- [ ] ... there are no open or closed issues that are related to my idea

### Description

I find that [lunr-languages](https://github.com/MihaiValentin/lunr-languages) supports Chinese. But when I try to configure "search" plugin in mkdocs, it fails with following error.
```
INFO     -  Building documentation...
INFO     -  Option search.lang 'zh' is not supported, falling back to 'en'
INFO     -  [00:44:06] Reloading browsers
```

Is there any plan to support Chinese search in mkdocs?

### Use Cases

mkdocs.yml
---
plugins:
  - search:
      lang: zh

**Comment by squidfunk** [comment]
Thanks for reporting. We can't use the Chinese language support because it uses native extensions. I've explained the problem in https://github.com/mkdocs/mkdocs/issues/2509#issuecomment-882689383. Until this is fixed in `lunr-languages`, Chinese search works somewhat with the Japanese language support, but it's not perfect.

**Comment by squidfunk** [comment]
Experimental support for Chinese segmentation has been added in https://github.com/squidfunk/mkdocs-material/issues/3882#issuecomment-1117554334.

**Comment by squidfunk** [comment]
Chinese language support has been released as part of [8.2.13+insiders-4.14.0](https://squidfunk.github.io/mkdocs-material/insiders/changelog/#4.14.0). The [blog article](https://squidfunk.github.io/mkdocs-material/blog/2022/chinese-search-support/) sums up my previous comment and explains how to set it up. I'm not proficient in Chinese, so I'm very interested in feedback!
